### ISSUE TITLE
Are you the developer of the #530 Server?

### ISSUE THREAD

**Issue by PtrStruct** [fatrolls/OpenSSL-MSVC-2019-Precompiled-Newest#1]
Hi, this GitHub was connected back to an old repo I found for a #530 Server, are you that developer?

**Comment by fatrolls** [comment]
yes i was working on runescape 2 private servers when I was young i made a 530 server rewrite in C#.. it never took off i just quit

**Comment by fatrolls** [comment]
here is all the repos containing my old server 

https://github.com/search?q=runescape-server-csharp&type=repositories

**Comment by PtrStruct** [comment]
I found your project super fascinating and I was wondering if you may have a Discord? I'd love to talk more networking with you, if you're at all interested.

**Comment by fatrolls** [comment]
I moved on to C++ now and focus on making hacks for games now. Sure you may add me on whatsapp if you want.

Btw I wasn't that good at C# when I made that Runescape server infact I just started with C# back then and I used a Java to C# Converter and then just did a bunch of googling to fix the issues to make the damn thing compile.

The maker of the original source you see was actually XenoRune the converter did a good job converting the code to look pretty in C#. At the time XenoRune was the nicest written server code for the time it was made by Luke132\Graham who was one of those skilled Java Guru's who just sit on the forums and never make anything.

I would also look at Omega 525 source if you want to improve on the server here's a link: https://www.rune-server.ee/runescape-development/rs-503-client-server/projects/239901-omega-525-most-advanced-100-free-hd-private-server-date.html

WhatsApp: +1 (929) 324-0148
